week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,14 +1,15 @@
 // Add comments to explain what this function does. You're meant to use Google!
 function getNumber() {
     return Math.random() * 10;
-}
+}//returns a random integer between 0 and 9
 
 // Add comments to explain what this function does. You're meant to use Google!
 function s(w1, w2) {
     return w1.concat(w2);
-}
+}//returns a new string containing the text of the joined w1 and w2.
 
 function concatenate(firstWord, secondWord, thirdWord) {
+    return firstWord.concat(secondWord.concat(thirdWord))
     // Write the body of this function to concatenate three words together
     // Look at the test case below to understand what to expect in return 
 }
```

Comment:
your solution is correct. You can also simplify it by just adding more arguments to concat
like
```firstWord.concat(" ", secondWord, " ", thirdWord)```